### DISCUSSION TITLE
English Desktop Version?

### DISCUSSION THREAD

**Discussion by kevinwells22** [SWHL/RapidVideOCR#35] [Q&A]
Hi, this looks an amazing application, is there an English Desktop Version please?

**Comment by SWHL** [comment]
Do you want to recognize the English subtitiles by RapidVideOCR?

**Reply by kevinwells22** [reply]
I would like to use the desktop appliacation (RapidVideOCR-0.0.4-macos-12-X64-no-installer.zip), or windows version, I only read English, so would like an English version, with English buttons and descriptions please.

**Reply by kevinwells22** [reply]
I am happy to help with the translations, and as a Pyhton developer, I can also assist with the English version if required.

**Reply by SWHL** [reply]
If you can help create an English version of RapidVideoOCR Desktop, it is very welcome.
Your pull request is very welcome.
